app/build.gradle.kts

```diff
@@ -67,6 +67,11 @@ android {
         create("beta") {
             initWith(buildTypes.getByName("release"))
             applicationIdSuffix = ".beta"
+            isMinifyEnabled = true
```

Comment:
At this point should we just remove the `initWith` and just add `isDebuggable = false` here?

Reply:
Indeed!